app.py

```diff
@@ -197,7 +197,7 @@ def snap_details(snap_name):
 @app.route('/account')
 @login_required
 def get_account():
-    return publisher_views.get_account()
+    return publisher_views.get_account(LOGIN_URL)
```

Comment:
could you add instead a function like this (see [doc](http://flask.pocoo.org/docs/0.12/templating/#context-processors)):

```python
@app.context_processor
def inject_login():
  return dict(LOGIN_URL=LOGIN_URL)
```

I am pretty sure we will use this URL in other context (login button for example)

Reply:
done